e2e-tests/tools/common-config/index.js

```diff
@@ -67,7 +67,7 @@ commonConfig.configure = function (promises) {
   else if (process.env.CHROME_HEADLESS || process.env.TRAVIS) {
     console.log('-- Using Chrome Headless --');
     config.capabilities.chromeOptions = {
-      args: ['--headless','--disable-gpu','--window-size=1600x1200']
+      args: ['--headless','--disable-gpu','--window-size=1600x1200', '--no-sandbox']
```

Comment:
This seems like a crazy workaround - and it is highly recommended to **not** use `--no-sandbox`.

What is causing the browser to crash? There must be another underlying issue here.